packages/api/lambdas/sf-event-sqs-to-db-records.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 const get = require('lodash/get');
-const { parseSQSMessageBody } = require('@cumulus/aws-client/SQS');
+const { parseSQSMessageBody, sendSQSMessage } = require('@cumulus/aws-client/SQS');
```

Comment:
The changes to this file are intended to change this lambda's behavior to explicitly throw failed messages to the SQS/Lambda DLQ in the case that writing the execution, pdr or granule to the database fails.      The lambda will now complete all writes explicitly before evaluating failure.